sql/001_create_volunteers.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `volunteers` (
+  `volunteer_id` int NOT NULL,
```

Comment:
I suggest consistent naming for primary keys - sometimes it is `volunteer_id`, sometimes it is `c_id`, sometimes it is `idparents`. It is confusing. I suggest using just `id` for the primary key name - and using `auto_increment` keyword in the field definition so people won't have to figure out the right value to insert there.